.github/workflows/ci.yml

```diff
@@ -92,9 +96,11 @@ jobs:
       - name: dpcpp installation
         run: |
           # This CI system yields oneAPI dependencies from the oneDAL repository
-          bash ./oneDAL/.ci/env/apt.sh dpcpp
+          bash .ci/env/apt.sh dpcpp
```

Comment:
Doesn't it worth update comment for this command then?

Reply:
Good question @samir-nasibli, its still using the oneAPI dependencies from oneDAL, but its now using an artifact, as the main repo could be updated but the nightly build hasn't seen it yet, so the original comment is still valid.